lib/config/options/index.ts

```diff
@@ -1188,6 +1188,15 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
+  {
+    name: 'autoReplaceRegexGlobalMatch',
+    description:
+      'Use the control whether replacement regular expressions are global matches or only the first match.',
+    type: 'boolean',
+    stage: 'branch',
+    parent: 'packageRules',
```

Comment:
Are you sure you wouldn't want to just turn it off completely? i.e. remove this `parent` line

Reply:
Happy to follow your // @viceice advice here.  I left it this way so that you had the choice to either turn it off completely, or, specify it within a `packageRule` for very specific replacement scenarios (eg: like those shared in the description of https://github.com/renovatebot/renovate/issues/20575)

Reply:
Setting parent=packageRules mean you can _only_ configure it within packageRules, so best to remove this

Reply:
removed parent=packageRules